bankgreen/settings.py

```diff
@@ -142,3 +144,5 @@
 USERNAME = os.environ.get("USERNAME")
 TOKEN = os.environ.get("TOKEN")
 PASSWORD = os.environ.get("PASSWORD")
+
+CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
Please put this as an env import. I don't like having it in the main settings.py that will be pulled on the live site.

Reply:
done